extensions/wikia/LookupContribs/templates/main-form.tmpl.php

```diff
@@ -58,9 +58,9 @@ function lcShowDetails(dbname, mode) {
 			{ "bVisible": true,  "aTargets": [ 2 ], "bSortable" : true, "sClass": "lc-datetime" },
 			{
 				"fnRender": function ( oObj ) {
-					var row = '<span class="lc-row"><a href="' + oObj.aData[3] + '">' + oObj.aData[3] + '</a></span>';
-					row += '&nbsp;(<a href="' + oObj.aData[3] + 'index.php?title=Special:Contributions/' + username + '">';
-					row += '<?=wfMsg('lookupcontribscontribs')?>';
+					var row = '<span class="lc-row"><a href="' + mw.html.escape( oObj.aData[3] ) + '">' + mw.html.escape( oObj.aData[3] ) + '</a></span>';
+					row += '&nbsp;(<a href="' + mw.html.escape( oObj.aData[3] ) + 'index.php?title=Special:Contributions/' + mw.html.escape( username ) + '">';
```

Comment:
Since this is a URL parameter, might be best to `mw.html.escape( encodeURIComponent( username ) )` for correctness of the URL.